### ISSUE TITLE
192.168.0.102 | Can't prepare gateway

### ISSUE THREAD

**Issue by ctrlands** [AlexxIT/XiaomiGateway3#860]
Multi-Mode Gateway 2 (V1.0.3_0028)
![image](https://user-images.githubusercontent.com/29942027/200983900-26578010-ecba-4436-b546-4c1701ec58e3.png)
![image](https://user-images.githubusercontent.com/29942027/200984216-0d1b7151-7326-4b56-99df-36d67b1e658c.png)
![image](https://user-images.githubusercontent.com/29942027/200984062-ec10cf0e-8d07-47c6-b10d-a33d835f239d.png)

**Comment by th3weird0** [comment]
same issue for me ;(

**Comment by macher91** [comment]
Use root user if you want use telnet

**Comment by AlexxIT** [comment]
This is wrong telnet cmd for MMG2. Use integration master version with empty telnet cmd. The right one is built-in.

**Comment by ctrlands** [comment]
> same issue for me ;(

@th3weird0 you can find the answer in #822

**Comment by ctrlands** [comment]
> This is wrong telnet cmd for MMG2. Use integration master version with empty telnet cmd. The right one is built-in.

@AlexxIT thks, looking forward to your great works
